src/enum_net.cpp

```diff
@@ -1052,7 +1052,7 @@ namespace libtorrent {
 		nl_msg->nlmsg_len = NLMSG_LENGTH(sizeof(rtmsg));
 		nl_msg->nlmsg_type = RTM_GETROUTE;
 		nl_msg->nlmsg_flags = NLM_F_DUMP | NLM_F_REQUEST;
-		nl_msg->nlmsg_seq = seq++;
+		nl_msg->nlmsg_seq = seq;
```

Comment:
is ``seq`` a constant now? or is it passed in by reference to ``read_nl_sock``?
if it is, would you mind making it ``const``?
I would also appreciate if you could add some comments about what you've learned about what ``seq`` and ``pid`` do